packages/webapp/pages/feeds/[slugOrId]/edit.tsx

```diff
@@ -0,0 +1,234 @@
+  const { showPrompt } = usePrompt();
+  const { FeedPageLayoutComponent } = useFeedLayout();
+
+  const feedSlugOrId = router.query.slugOrId as string;
+  const feed = useMemo(() => {
+    return feeds?.edges.find(
+      (edge) =>
+        edge.node.slug === feedSlugOrId || edge.node.id === feedSlugOrId,
+    )?.node;
+  }, [feeds, feedSlugOrId]);
+  const feedId = feed?.id;
+
+  const { user } = useAuthContext();
+  const { feedSettings } = useFeedSettings({ feedId, enabled: !!feedId });
+  const [isPreviewFeedVisible, setPreviewFeedVisible] = useState(true);
+  const isPreviewFeedEnabled = feedSettings?.includeTags?.length >= 1;
+  const [isDirty, setDirty] = useState(false);
+
+  const onValidateAction = () => {
+    return !isDirty;
+  };
+
+  const { onAskConfirmation } = useExitConfirmation({
+    message: discardPrompt.description as string,
+    onValidateAction,
+  });
+
+  const { mutateAsync: onSubmit, isLoading } = useMutation(
+    async ({ name }: EditFeedFormProps) => {
+      const result = await updateFeed({ feedId, name });
+
+      await request(graphqlUrl, ADD_FILTERS_TO_FEED_MUTATION, {
+        feedId: result.id,
+        filters: {
+          includeTags: feedSettings?.includeTags || [],
+        },
+      });
+
+      return result;
+    },
+    {
+      onSuccess: (data) => {
+        queryClient.removeQueries(getFeedSettingsQueryKey(user, feedId));
+
+        onAskConfirmation(false);
+        router.replace(`${webappUrl}feeds/${data.id}`);
+      },
+      onError: () => {
+        displayToast(labels.error.generic);
+      },
+    },
+  );
+
+  const { mutateAsync: onDelete, status: deleteStatus } = useMutation(
+    async () => {
+      const shouldDelete = await showPrompt({
+        title: `Delete ${feed?.flags?.name || 'custom feed'}?`,
+        description: labels.feed.prompt.delete.description,
+        okButton: {
+          title: labels.feed.prompt.delete.okButton,
+          color: ButtonColor.Ketchup,
+        },
+      });
+
+      if (!shouldDelete) {
+        throw new Error('User cancelled deletion');
+      }
+
+      const result = await deleteFeed({ feedId });
+
+      return result;
+    },
+    {
+      onSuccess: () => {
+        queryClient.removeQueries(getFeedSettingsQueryKey(user, feedId));
+
+        onAskConfirmation(false);
+        router.replace('/');
+      },
+    },
+  );
+
+  const shouldRedirectToNewFeed =
+    feeds && feedSlugOrId && deleteStatus === 'idle';
+
+  useEffect(() => {
+    if (!shouldRedirectToNewFeed) {
+      return;
+    }
+
+    if (!feed) {
+      router.push(`${webappUrl}feeds/new`);
+    }
+  }, [shouldRedirectToNewFeed, feed, router]);
+
+  if (!feed) {
+    return null;
+  }
+
+  const seo: NextSeoProps = {
+    title: 'Edit custom feed',
+    openGraph: { ...defaultOpenGraph },
+    ...defaultSeo,
+  };
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <FeedPageLayoutComponent className={classNames('!p-0')}>
```

Comment:
```suggestion
      <FeedPageLayoutComponent className='!p-0'>
```
Also wondering why you need the override here?